sections/collage.liquid

```diff
@@ -56,31 +64,25 @@
                     -%}
                     {%- if large_block -%}
                       {%- capture sizes -%}
-                        {% if section.blocks.size == 1 -%}(min-width: {{ settings.page_width }}px) calc({{ settings.page_width }}px - 100px){% else %}(min-width: {{ settings.page_width }}px) calc(({{ settings.page_width }}px - 100px) * 2 / 3 - {{ grid_space_desktop }}){% endif %},
-                        {% if section.blocks.size == 1 -%}(min-width: 750px) calc(100vw - 100px){% else %}(min-width: 750px) calc((100vw - 100px) * 2 / 3 - {{ grid_space_desktop }}){% endif %},
-                        {% if section.blocks.size == 2 and section.settings.mobile_layout == 'collage' -%}calc((100vw - 30px) / 2){% else %}calc(100vw - 30px){% endif %}
+                        {% if section.blocks.size == 1 -%}(min-width: {{ settings.page_width }}px) calc({{ settings.page_width | times: size_multiplier }}px - 100px){% else %}(min-width: {{ settings.page_width }}px) calc(({{ settings.page_width | times: size_multiplier }}px - 100px) * 2 / 3 - {{ grid_space_desktop }}){% endif %},
+                        {% if section.blocks.size == 1 -%}(min-width: 750px) calc({{ 100 | times: size_multiplier }}vw - 100px){% else %}(min-width: 750px) calc(({{ 100 | times: size_multiplier }}vw - 100px) * 2 / 3 - {{ grid_space_desktop }}){% endif %},
+                        {% if section.blocks.size == 2 and section.settings.mobile_layout == 'collage' -%}calc(({{ 100 | times: size_multiplier }}vw - 30px) / 2){% else %}calc({{ 100 | times: size_multiplier }}vw - 30px){% endif %}
                         {%- endcapture -%}
                       {{
                         block.settings.image
-                        | image_url: width: 3200
-                        | image_tag:
-                          loading: 'lazy',
-                          sizes: sizes,
-                          widths: '50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3200'
+                        | image_url: width: 3840
+                        | image_tag: loading: 'lazy', sizes: sizes, widths: widths
                       }}
                     {%- else -%}
                       {%- capture sizes -%}
-                           (min-width: {{ settings.page_width }}px) calc(({{ settings.page_width }}px - 100px) * 1 / 3 - {{ grid_space_desktop }}),
-                           (min-width: 750px) calc((100vw - 100px) * 1 / 3 - {{ grid_space_desktop }}),
-                           {% if section.settings.mobile_layout == 'collage' %}calc((100vw - 30px) / 2 - {{ grid_space_mobile }}){% else %}calc(100vw - 30px){% endif %}
+                           (min-width: {{ settings.page_width }}px) calc(({{ settings.page_width | times: size_multiplier }}px - 100px) * 1 / 3 - {{ grid_space_desktop }}),
```

Comment:
Getting some wonky sizes loading with these changes on collage. Haven't seen the issue in the other sections yet.
<img src="https://screenshot.click/24-53-w5xx1-mflt2.png" width="400" />

Reply:
Tho if you look at the network tab in the inspect tool, it's loading the expected image size right ? 

We're basing our `sizes` attribute values on what we already had by multiplied by `1.2`. So the images we're getting should be a bit bigger than their containers. 

There is also a PR from folks in theme support to fix an issue with the image logic in that section: https://github.com/Shopify/dawn/pull/2277